protected/humhub/modules/user/commands/UserController.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+/**
+ * @link https://www.humhub.org/
+ * @copyright Copyright (c) 2020 HumHub GmbH & Co. KG
+ * @license https://www.humhub.com/licences
+ */
+
+namespace humhub\modules\installer\commands;
```

Comment:
Also you should make sure to change the namespace to match the user core module :+1:

Reply:
Thanks @ArchBlood